dashboard/app/controllers/lessons_controller.rb

```diff
@@ -44,7 +53,7 @@ def student_lesson_plan
     @lesson_data = @lesson.summarize_for_student_lesson_plan
   end
 
-  # GET /lessons/1/edit
+  # GET /lessons/1/edit where 1 can be the id or the lesson position
```

Comment:
this is a bit confusing. I would think about things in this order:

1. did you manage to make it so that /s/script/lessons/1/edit will allow you to edit the lesson? if so, let's have this be the only way to edit the lesson.

2. if that option doesn't work and we do need two routes, then what you're describing on this line to overload the controller action is a bit too confusing. I would suggest sending this new route to a different controller action, such as `edit_script_lesson`.

Reply:
Option 1 doesn't work because you need a way to edit lessons without lesson plans and those don't have the lesson position URL. 

I can look at option 2.